lib/AutoTypeX11.cpp: In member function ‘void AutoTypeX11::sleepTime(int)’:
lib/AutoTypeX11.cpp:202:30: error: ‘nanosleep’ was not declared in this scope

--- origsrc/keepassx-0.4.3/src/lib/AutoTypeX11.cpp	2009-09-13 09:43:10.000000000 -0500
+++ src/keepassx-0.4.3/src/lib/AutoTypeX11.cpp	2011-05-16 22:51:36.924809400 -0500
@@ -22,6 +22,7 @@
 
 #include "mainwindow.h"
 #include "lib/HelperX11.h"
+#include <ctime>
 #include <QX11Info>
 
 #ifndef GLOBAL_AUTOTYPE
